packages/ui/src/assets/css/tooltip.css

```diff
@@ -1,6 +1,51 @@
 .tooltip-container {
+  background-color: #555;
```

Comment:
uss css variable for the color and reuse it in below code for the borders. This way, they are always same and it will be easier to change the color from just one place.